package-lock.json

```diff
@@ -1,13624 +0,0 @@
-{
```

Comment:
Why did we delete the lock file? It's hard to keep our node_module folder consistent without one and github actiona won't run

Reply:
Jus an accident. Nice catch